src/http_server.cc

```diff
@@ -2421,6 +2419,7 @@ HTTPAPIServer::EVBufferToInput(
               "Unable to parse 'data'");
 
           if (dtype == TRITONSERVER_TYPE_BYTES) {
+            byte_size = 0;
```

Comment:
Can you add a comment on why this initialization is important here?

Reply:
Yes, added.